static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -150,6 +150,23 @@ define([
         });
     }
 
+    function initHeroic(){
+        if($('.vjs-tech') && $('.labour-liverpool-headline') && $('.vjs-control-bar')){
+            var player = $('.vjs-tech');
+            var headline = $('.labour-liverpool-headline');
+            var controls = $('.vjs-control-bar');
+
+            bean.on(player[0], 'playing', function () {
+                bonzo(headline[0]).addClass('playing');
```

Comment:
I think headline should be a bonzo object already so you should be able to do `headline.addClass` - same below